Add Control Center documentation (#450)
netbirdio/docs: src/pages/how-to/control-center.mdx

## Views

### Peers view

Use this to understand what a specific machine can reach.

- Click the peer node, then search or choose another peer from the dropdown to switch focus.
- The graph shows the peer's access control policy nodes and the reachable resources.
- Click a policy chip to open the standard policy editor. Changes you save are reflected in the graph immediately.

Tips:

- Great for incident review or validating a new server's least-privilege access.

### Groups view

Use this to validate team-level entitlements.

<p>
    <img src="/docs-static/img/how-to-guides/control-center-groups-view.png" alt="Control Center Groups view showing DevOps Berlin group connections" className="imagewrapper-big"/>
</p>

- Click a group node, then search or choose from the dropdown to switch groups.
- The layout shows which resources that group can reach and via which policies.
- View-only for topology here; create or delete groups in the Groups section outside Control Center. Group-based access is the recommended way to manage permissions.

Common checks:

- Confirm that "DevOps" can reach RDS on TCP 5432, or that "Support" only reaches SSH on TCP 22.

### Networks view

> Use this to see who can access your routed networks and which ports are open.

Reviewer: ```suggestion
Use this to see who can access resources in your routed networks.
```
